Plugins/OpenAPIGeneratorCommand/plugin.swift

```diff
@@ -0,0 +1,137 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftOpenAPIGenerator open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import PackagePlugin
+import Foundation
+
+@main
+struct SwiftOpenAPIGeneratorPlugin {
+    func runCommand(
+        targetWorkingDirectory: Path,
+        tool: (String) throws -> PluginContext.Tool,
+        sourceFiles: FileList,
+        targetName: String
+    ) throws {
+        let inputs = try PluginUtils.validateInputs(
+            workingDirectory: targetWorkingDirectory,
+            tool: tool,
+            sourceFiles: sourceFiles,
+            targetName: targetName,
+            invocationSource: .CommandPlugin
+        )
+
+        let toolUrl = URL(fileURLWithPath: inputs.tool.path.string)
+        let process = Process()
+        process.executableURL = toolUrl
+        process.arguments = inputs.arguments
```

Comment:
Can you also clean the environment, to ensure that nothing leaks through. If that breaks the plugin, only add the exact env vars required.
```suggestion
        process.arguments = inputs.arguments
        process.environment = [:]
```